lib/jsdom/browser/default-stylesheet.js

```diff
@@ -173,36 +160,12 @@ h1 {
     font-weight: bold
 }
 
-:-webkit-any(article,aside,nav,section) h1 {
+article h1, aside h1, nav h1, section h1 {
     font-size: 1.5em;
     -webkit-margin-before: 0.83__qem;
     -webkit-margin-after: 0.83em;
 }
 
-:-webkit-any(article,aside,nav,section) :-webkit-any(article,aside,nav,section) h1 {
```

Comment:
Nope, `:matches()` is CSS4, and [nwmatcher only supports CSS3](https://github.com/dperini/nwmatcher/wiki/CSS-supported-selectors#content-pseudo-element-selectors-not-supported):

>The list has been built upon current W3C Selectors Level 3, but differs in that it contains some deprecated specification that we needed to temporarily maintain.

Going through the source code confirms that `:matches()` is not implemented.

> If not, we should add a comment here explaining that we are going against spec for now.

Done!